substrate/client/network/src/request_responses.rs

```diff
@@ -248,6 +284,12 @@ pub enum Event {
 		/// Reputation changes.
 		changes: Vec<ReputationChange>,
 	},
+
+	/*CustomOutboundFailure {
+		peer: PeerId,
+        request_id: OutboundRequestId,
+        error: CustomOutboundFailure,
```

Comment:
Why?

Reply:
Trying to replace the libp2p dependent event found on lines `836~840`...